lib/utils/html-tag.html

```diff
@@ -12,13 +12,31 @@
   (function() {
     'use strict';
 
+    class HTMLLiteral {
+      constructor(string) {
+        this.innerHTML = string;
```

Comment:
I wouldn't name this `innerHTML`, this isn't an element. I'd go with `value`

Reply:
It was an attempt to write less code, since both types can return their string value via `innerHTML`; fair that it's a bit short-cutty.